### ISSUE TITLE
“Deprecation: Document#title is now a key in the #data hash.”

### ISSUE THREAD

**Issue by BorisAnthony** [maximevaillancourt/digital-garden-jekyll-template#10]
Salut Maxime!
Just pulled your update. Great additions thank you. 👍 

I run the latest Jekyll main branch and got the above error.

It is a simple fix but I am not set up right now to easily submit a Pull Request so forgive me for just sharing the fix here. 

bidirectional_links_generator.rb
ln 34:
`label: current_note.title,`
to
`label: current_note.data['title'],`

&
ln 56:
`note.title.to_i(36).to_s`
to
`note.data['title'].to_i(36).to_s`

**Comment by BorisAnthony** [comment]
from my Gemfile
```
#gem "jekyll", "~> 4.0.0"
gem "jekyll", github: "jekyll/jekyll"
```

**Comment by maximevaillancourt** [comment]
Fixed in a63a231. Thank you Boris! 🙌
